### ISSUE TITLE
New-SSHSession -OperationTimeout and -ConnectionTimeout

### ISSUE THREAD

**Issue by dbrennand** [darkoperator/Posh-SSH#323]
Hi,

For the `New-SSHSession` function. It isn't documented if `OperationTimeout` or `ConnectionTimeout` are in seconds or milliseconds.
Without the function it errors saying the host didn't respond in `10000` milliseconds. However, I just wanted to ask to confirm that it's milliseconds?
@darkoperator

**Comment by darkoperator** [comment]
Seconds https://github.com/darkoperator/Posh-SSH/blob/master/Source/PoshSSH/PoshSSH/NewSessionBase.cs#L139 <https://github.com/darkoperator/Posh-SSH/blob/master/Source/PoshSSH/PoshSSH/NewSessionBase.cs#L139> need to figure out why the help info is not shown

**Comment by dbrennand** [comment]
Hmmm, it doesn't seem to be waiting the time I specify. 🤔

**Comment by dbrennand** [comment]
In the code you linked, I cannot see where the -OperationTimeout parameter is used.

**Comment by darkoperator** [comment]
You are correct, will remove operation timeout. The connection info object does not have a property to set it, seems it is a copy paste mistake from another cmdlet.

**Comment by L-P-G** [comment]
@darkoperator is connection timeout also affected by this? I'm seeing Set-SCPFile failing to error out when using connection or operationtimeout. I can open a new issue if need be.

**Comment by ckaminski** [comment]
Is it just me, or in NewSessionBase.cs is OperationTimeout not getting set?

See roughly line 344 where the connection timeout is set.. I see no usage of operation timeout on the scpclient object (since it's not a property on the baseclient?)

I assume the other clients in the switch on line 253 suffer the same?
